src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue16175.cs

```diff
@@ -0,0 +1,27 @@
+﻿#if IOS || MACCATALYST // More tab is available in MAC and iOS
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues;
+
+public class Issue16175 : _IssuesUITest
+{
+	public Issue16175(TestDevice testDevice) : base(testDevice)
+	{
+	}
+	public override string Issue => "OnNavigatedTo was not triggered when tabs in More section";
+
+	[Test]
+	[Category(UITestCategories.TabbedPage)]
+	public void TabbedPageTabEventsTriggeredInMoreSection()
```

Comment:
The test is failing on iOS:
`The app was expected to be running still, investigate as possible crash`
Seems like the App is crashing.

Reply:
@jsuarezruiz I have verified it again, and the test still passes locally without any issues. However, I added an additional WaitForElement when the page appears. Could you please re-run the CI to check the results?